docs/administration.md

```diff
@@ -232,7 +232,7 @@ Note: This will only work when the replication factor is >= 2
   - List of all indexes on your cluster
   - List of all frames in your indexes
   - Max slice per index, listed in the `/slices/max` endpoint
-- With this information you can query the `/fragment/nodes` endpoint and iterate over each slice
+- With this information you can query the `/internal/fragment/nodes` endpoint and iterate over each slice
```

Comment:
iterate over each *shard*

Reply:
@yuce I didn't make all the necessary changes to `docs`. We're going to address that all at once.